README.md

```diff
@@ -2,31 +2,102 @@
 
-## Conclusion
+We see IntelliJ warning us of an error. When we try to run the program, Java
+will first try to compile the source code. But it stops before it can even
+proceed running it:
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+![build-error](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/intellij-compile-time-error.png)
 
-## Resources
+We see that the build failed. This means, it failed to compile the code.
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+Some examples of compile-time errors include:
+
+- Missing parentheses and curly brackets.
+
+  ![missing-curly-bracket](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/compile-time-error-missing-curly-bracket.png)
+
+- Missing terminator.
+
+  ![missing-semicolon](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/compile-time-error-missing-terminator.png)
+
+- And other syntax type errors.
+
+  ![syntax-errors](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/compile-time-error-syntax-errors.png)
+
+## What are Runtime Errors?
+
+**Runtime errors** are errors that occur when we try to execute the program.
+These errors can be compiled successfully, but will occur when we are running
+the code. So, if we were using a terminal window, these errors would show up
+once we run the `java` command on the compiled code.
+
+We can see these errors when we try to run our programs in IntelliJ too. A
+common runtime error is a division by zero. As we may recall from math class,
+we cannot divide another number by zero, as it results in a "not a number".
+
+Let's see what happens when we run this code:
+
+```java
+public class Main {
+    public static void main(String[] args) {
+        int notANumber = 9 / 0;
+        System.out.println(notANumber);
+    }
+}
+```
+
+There is nothing wrong with the syntax of the code, so the compiler will be able
+to compile it into Java bytecode successfully. But the issue here is that 9 is
+being divided by 0, which results in not a number. IntelliJ even warns us if we
+hover over the expression that there could be something wrong here:
+
+![numeric-overflow-in-expression](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/intellij-warn-division-by-zero.png)
+
+If we ignore IntelliJ and try to run it, notice again that this gets past the
+compilation step and throws the exception at runtime:
+
+![arithmeticexception](https://curriculum-content.s3.amazonaws.com/java-mod-1/errors/runtime-error-divide-by-zero.png)
+
+Notice in this example that the program compiled and was able to start running
+successfully. It wasn't until the program tried to execute the
+`int notANumber = 9 / 0;` that the program failed to continue executing. This is
+an example of a runtime error.
+
+Notice in this example that the program compiled and was able to start running
```

Comment:
Is an example image missing here?

Reply:
Oops! This was leftover when I refactored this. Thanks for catching. I removed it!